### DISCUSSION TITLE
The system is giving an error when installing in docker

### DISCUSSION THREAD

**Discussion by miguel2962** [microrealestate/microrealestate#193] [Ideas]
After the last changes I created a new server to test the reset password part because it is not working for me... and I could not install it due to new errors with the emailer error part...

Another suggestion would be to add that the admin can reset the passwords of the collaborators..

**Comment by camelaissani** [comment]
@miguel2962 can you give more details about where it is not working? Any logs?

**Reply by miguel2962** [reply]
I just tried a clean install and it installed fine now... I would like to update the system on my server without doing a clean install and losing my data... do you know how I could do that with Microrealstate?

**Reply by camelaissani** [reply]
You can follow this https://github.com/microrealestate/microrealestate/discussions/141
Replace `./mre` command by `yarn run mre`
